### DISCUSSION TITLE
askama.toml is placed correctly?

### DISCUSSION THREAD

**Discussion by Ken-Miura** [tokio-rs/axum#203] [Q&A] [answered]
All examples were moved in the following commit. And now that each example has Cargo.toml
https://github.com/tokio-rs/axum/commit/6c9651c14af9d0e3fafda5f5187a7747f748003e

According to the document (https://djc.github.io/askama/configuration.html), It seems that askama.toml should be placed at the directory where Cargo.toml is placed.

As of now on main branch (commit id: e22045d42f91757f7224bdad35385e1119d5bfbe), askama.toml is placed at root of the project.

askama.toml is still placed correctly?

I'm not familiar with askama, so this might be silly question. If so, please just close this discussion.

**Comment by davidpdrsn** [comment, marked as answer]
Good catch! It was still working, but not necessary actually. Removed in https://github.com/tokio-rs/axum/commit/5368e33f99d589c889e075c1375b45f13f17fc7e
